packages/gridstack/src/components/ConfigurableDashboard/ConfigurableDashboard.vue

```diff
@@ -94,6 +97,12 @@ const onAddWidget = async (widgetKey: string) => {
       ?.scrollIntoView({ behavior: 'smooth' })
   }, 250)
 }
+
+watch(editMode, (isEditMode) => {
+  if (isEditMode) {
+    currentConfig.value = dashboardConfig.value
+  }
+})
```

Comment:
Did we not already set the currentConfig on another place? I'm not a big fan of watcher if we can solve this with event handler.

Reply:
The problem is when the dashboardConfig is changed from the select input. As we are now using defineModel, so we're just making the change on dashboardConfig. So, we would either way have to have a watcher somewhere. But DashboardConfig is also changed itself on the editmode, so I didn't want to track the change of dashboard config but rather the start of the editMode.